www/source/blog/2018-02-21-Habitat-Helm.html.md

```diff
@@ -57,24 +44,24 @@ Once you've cloned core-plans locally, go to `core-plans/nginx` and enter the Ha
 
 * Enter the Habitat Studio within the nginx core plan with `hab studio enter`
 * Build the latest nginx package and stores it as a habitat artifact (“hart”) in the nginx plan’s `results` directory: `build`
-* Export the hab artifact to a Helm chart: `hab pkg export helm results/<yourfilename>.hart` This command will look very similar to `hab pkg export helm results/tdrew-nginx-1.11.10-20180221053714-x86_64-linux.hart`
+* Initialize the Helm client-side: `helm init -c`
+* Export the hab artifact to a Helm chart: `hab pkg export helm --download-deps results/<yourfilename>.hart` This command will look very similar to `hab pkg export helm --download-deps results/tdrew-nginx-1.11.10-20180221053714-x86_64-linux.hart`
```

Comment:
I would put a new line here before the commands, to make it clearer.

Reply:
on a separate para you mean?